source/site-administrators/studio/using-the-proxy-config.rst

```diff
@@ -0,0 +1,161 @@
+
+       .. code-block:: text
+          :caption: *video-center-blueprint/sources/app/.env.development*
+
+          REACT_APP_BASE_URL=http://localhost:8080
+          WDS_SOCKET_PORT=3000
+
+#. Setup Studio
+
+   * Create a site using the video center blueprint from the Public Marketplace.
+
+     From the **Main Menu**, click on **Sites**, then click on the **Create Site** button.  This will open the **Create Site** dialog.  Click on the **Public Marketplace** tab.  Look for **Video Center**, then click on the **Use** button, fill in the required information then click on the **Review** button, then finally the **Create Site** button.  This **Video Center** blueprint we selected from the Marketplace is the same react application
+
+     .. image:: /_static/images/site-admin/vcbp-marketplace.jpg
+        :alt: Select Video Center blueprint from the Public Marketplace
+        :width: 70 %
+        :align: center
+
+     |
+
+
+   * Setup the proxy for the video center React application we started above
+
+     Open the **Sidebar**, click on |siteConfig|, then click on **Configuration**.  Select **Proxy Config** from the dropdown menu.
+
+     .. image:: /_static/images/site-admin/config-open-proxy-config.jpg
+        :alt: Configurations - Open Proxy Configuration
+        :width: 55 %
+        :align: center
+
+     |
+
+     Scroll down to the ``preview`` server and edit the ``url`` to point to the url used for the React application (``localhost:3000``) we setup in the beginning.  Save your changes.
+
+       .. code-block:: xml
+          :caption: *CRAFTER_HOME/data/repos/sites/sandbox/SITENAME/sandbox/config/engine/proxy-config.xml*
+          :emphasize-lines: 3
+
+          <server>
+            <id>preview</id>
+            <url>http://localhost:3000</url>
+            <patterns>
+              <pattern>.*</pattern>
+            </patterns>
+          </server>
+
+     At this point, the preview we are seeing in Studio should be the one from our React application.
+
+   * Modify the React application then verify that we can preview the changes made inside Studio.
+
+     For this part, we'll change the text ``Featured Channels`` in the home page to ``My Featured Channels``.  Using your favorite editor, in your React app, navigate to ``video-center-blueprint/sources/app/src/containers/Home`` and open the ``Home.js`` file. Scroll down to the line with ``key: 'featured-channels'`` and edit the ``value``:
+
+       .. code-block:: js
+          :emphasize-lines: 3
+
+          {
+            key: 'featured-channels',
+            value: 'My Featured Channels',
+            type: 'channel-card-alt',
+            ...
+
+     Save your changes.  Notice that in the React app preview (localhost:3000), the page is reloaded with our changes now visible.  Now let's take a look at Studio.  Refresh the browser and notice that the changes we made in the React app is visible inside the Studio preview
```

Comment:
The refresh on Studio should not be necessary, the reload should be automatic otherwise the example is not working as expected